tests/generators/kzg_4844/main.py

```diff
@@ -0,0 +1,288 @@
+"""
```

Comment:
Should this file be split into many files, like one file per one kzg-4844 function? I think doing it similarly to the one in https://github.com/ethereum/consensus-specs/tree/dev/tests/generators/ssz_generic is more organized.

Reply:
No strong opinion on this one. @hwwhww what would you prefer? I copied from the bls generators which seem to have this structure

Reply:
I slightly prefer separate files, but I can see there are some shared helpers/constants. ~500 lines look acceptable so far, but if you expect to extend the test cases, refactoring them early seems better.

Reply:
Decided on the call today to leave for now and split when the file becomes too large.